HTML/Common.php

```diff
@@ -141,7 +141,7 @@ function _getAttrString($attributes)
         if (is_array($attributes)) {
             $charset = HTML_Common::charset();
             foreach ($attributes as $key => $value) {
-                $strAttr .= ' ' . $key . '="' . htmlspecialchars(($value ?? ''), ENT_COMPAT, $charset) . '"';
+                $strAttr .= ' ' . $key . '="' . htmlspecialchars((is_array($value) ? '' : $value ?? ''), ENT_COMPAT, $charset) . '"';
```

Comment:
@MegaphoneJon just for the sake of clarity are you able to put `(` and `)` around the $value ?? '' in the false part of the first ternary operator

Reply:
Sorry @seamuslee001, just saw this.